docs/DESIGN-SYSTEM.md

```
@@ -0,0 +1,443 @@
+
+**Scaffold location**: `repository-files/rust/docs/`
+
+| File | Purpose |
+|------|---------|
+| `README.md` | Documentation conventions guide |
+| `jbcom-rustdoc.css` | jbcom-branded rustdoc CSS |
+
+Usage:
+1. Write doc comments in source code (`///` and `//!`)
+2. Generate with custom CSS:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)